sklearn/metrics/cluster/_supervised.py

```diff
@@ -44,6 +45,12 @@ def check_clusterings(labels_true, labels_pred):
     labels_pred : array-like of shape (n_samples,)
         The predicted labels.
     """
+    if (type_of_target(labels_true) == "continuous" or
+            type_of_target(labels_pred) == "continuous"):
+        raise ValueError('Classification metrics expects discrete \
+            values received %r for label, and %r \
+            for target' % (labels_pred.dtype.type, labels_true.dtype.type))
```

Comment:
Is there a reason you're not using the [check_targets function](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/metrics/_classification.py#L54) from the classification module?

Reply:
No there is no particular reason for using that. As only categorical arrays are allowed type_of_target gets the job done.